services/user.py

```diff
@@ -0,0 +1,53 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None,
+
+) -> None:
+    user = get_user_model().objects.create_user(
+        username=username,
+        password=password,
+    )
```

Comment:
The `create_user` method is used correctly here. However, ensure that any exceptions raised during user creation, such as `IntegrityError` for duplicate usernames, are handled or documented.